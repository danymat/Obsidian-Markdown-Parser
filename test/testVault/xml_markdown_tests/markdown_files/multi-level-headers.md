Root level text.

# Header 1

Level 1 text.

## Header 2

Level 2 text.

### Header 3

Level 3 text.

###### Header 6

Level 6 text.

# Section 2

Another section.
