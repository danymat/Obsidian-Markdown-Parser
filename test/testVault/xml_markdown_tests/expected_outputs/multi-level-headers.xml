<root level="0">Root level text.

<section level="1" title="Header 1"># Header 1

Level 1 text.

<section level="2" title="Header 2">## Header 2

Level 2 text.

<section level="3" title="Header 3">### Header 3

Level 3 text.

<section level="6" title="Header 6">###### Header 6

Level 6 text.

</section></section></section></section><section level="1" title="Section 2"># Section 2

Another section.
</section></root>
